experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/ContainerCommand.scala

```diff
@@ -0,0 +1,58 @@
+package org.apache.gearpump.experiments.yarn.master
+
+import java.io.File
+
+import org.apache.gearpump.experiments.yarn.AppConfig
+import org.apache.gearpump.experiments.yarn.Constants._
+import org.apache.gearpump.transport.HostPort
+import org.apache.gearpump.util.Constants
+import org.apache.hadoop.yarn.api.ApplicationConstants
+
+
+abstract class ContainerCommand {
+  val appConfig: AppConfig
+  val version = appConfig.getEnv("version")
```

Comment:
This is GearPump's version as provided in the cliOpts not HOCON - should probably be either moved to a conf file or represented there. BTW although we have version.sbt we do not have a way of moving this into a HOCON representation by script or programmatically in scala or otherwise and probably should.